cmd/clusterctl/cmd/move.go

```diff
@@ -79,6 +95,32 @@ func runMove() error {
 		return err
 	}
 
+	// Only perform backup
+	if mo.backupToFolder {
+		if mo.dryRun {
+			return errors.New("please run --to-folder without --dry-run flag")
+		}
+
+		return c.Backup(client.BackupOptions{
```

Comment:
I will go a little bit further and deprecate Backup and Restore from the CLI replacing this calls with calls to Move.
This requires to add Directory to MoveOptions, and the document and handle various parameters combinations (we  need to do this twice because the client can be used without the CLI)

Reply:
I've removed the use of Backup and Restore in `cmd/move` and implemented the logic in `client/move` instead. I hope that I understood correctly. Once the logic is in the correct place and somewhat acceptable I will validate the flag combinations (dry run etc) work as expected.